### PR TITLE
docs(sui-studio): Adding basic styles for Demos standadisation

### PR DESCRIPTION
**Pull request by turolopezsanabria**
**What does this PR do?**
Add basic styles to standardise how the Demos should look like.

**Extra information:**
Nothing will change after publishing these changes, it's just a preparation for the Demos adjustment.

**This is how it looks like:**
styles preview

### CODE REVIEW

File: packages/sui-studio/src/components/layout/_style.scss [hunk 1]
```diff
@@ -40,7 +40,146 @@
       transform: translate3d(0, 0, 0);
     }
   }
-
+  .onehalf{
```
Comment (jcger) [inline packages/sui-studio/src/components/layout/_style.scss:43]: maybe a whitespace before each `{` ? 😅

Comment (turolopezsanabria) [reply packages/sui-studio/src/components/layout/_style.scss]: 👍 done, I've also removed .onehalf and .onethird, used only for my tests

**Comment by turolopezsanabria** [comment]
> there should be documentation in the readme
Docs in the SUI Studio readme giving visibility to these styles, right?
https://github.com/SUI-Components/sui/tree/master/packages/sui-studio

**Comment by turolopezsanabria** [comment]
Hey, we've just checked and these styles don't get applied in mode Dev. @andresin87 will move the styles to the correct file.

**Comment by andresin87** [comment]
> Hey, we've just checked and these styles don't get applied in mode Dev. @andresin87 will move the styles to the correct file.

done

**Review by andresin87 — APPROVED** [review]

**Pull request merged**